src/components/SystemPreferences.vue

```diff
@@ -39,6 +40,11 @@ export default {
       type:    Number,
       default: 0,
     },
+    // misc
+    telemetry: {
+      type:    Boolean,
+      default: false,
```

Comment:
#26 wants opt-out, not opt-in. (Personally I'd much prefer opt-in, of course, but that's a PM battle.)

Reply:
that tooltip is obsolete